scripts/init.sql

```diff
@@ -1,158 +1,147 @@
-        child_id integer NOT NULL,
-        item_id integer NOT NULL,
-        image_id integer NOT NULL,
-        order_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        CONSTRAINT wishcards_pkey PRIMARY KEY(id)
-    );
-
--- constraints
+SET check_function_bodies = false ;
+
+CREATE TABLE agencies(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    "name" varchar(255) NOT NULL,
+    website varchar(255) NOT NULL,
+    phone varchar(100) NOT NULL,
+    bio varchar(500),
+    address_line_1 varchar(255) NOT NULL,
+    address_line_2 varchar(255),
+    city varchar(100) NOT NULL,
+    state varchar(100) NOT NULL,
+    country varchar(100) NOT NULL,
+    zip_code varchar(50) NOT NULL,
+    verified boolean NOT NULL DEFAULT false,
+    employer_identification_number varchar(255) NOT NULL,
+    account_manager_id uuid NOT NULL,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamp,
+    CONSTRAINT agencies_pkey PRIMARY KEY(id),
+    UNIQUE(employer_identification_number)
+);
+
+CREATE TABLE children(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    first_name varchar(255) NOT NULL,
+    last_name varchar(255) NOT NULL,
+    birth_year integer NOT NULL,
+    interest varchar(255) NOT NULL,
+    story varchar(255) NOT NULL,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT children_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE community_posts(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    message varchar(255) NOT NULL,
+    agency_id uuid NOT NULL,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT community_posts_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE images(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    url varchar(255) NOT NULL,
+    meta_data json,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT images_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE items(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    "name" varchar(255) NOT NULL,
+    price numeric(10, 2) NOT NULL,
+    link varchar(2000) NOT NULL,
+    retailer varchar(255) NOT NULL,
+    retailer_product_id varchar(255) NOT NULL,
+    meta_data json,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT items_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE messages(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    "content" varchar(500) NOT NULL,
+    sender uuid NOT NULL,
+    wishcard_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT messages_pkey PRIMARY KEY(id)
+);
+
+CREATE TABLE orders(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    status integer NOT NULL DEFAULT 0,
+    delivery_date timestamp,
+    tracking_info varchar(500),
+    donor_id uuid NOT NULL,
```

Comment:
```suggestion
    donor_id uuid NOT NULL REFERENCES users (id),
```

added user foreign key constraint